integration-tests/cucumber-tests-platform-common/src/test/java/org/opensmartgridplatform/cucumber/platform/common/glue/steps/database/ws/CoreResponseUrlDataRepository.java

```diff
@@ -12,5 +12,5 @@
 import org.opensmartgridplatform.adapter.ws.domain.repositories.ResponseUrlDataRepository;
 import org.springframework.stereotype.Repository;
 
-@Repository
+@Repository("wsCoreResponseUrlDataRepository")
 public interface CoreResponseUrlDataRepository extends ResponseUrlDataRepository {}
```

Comment:
Heb je overwogen om de naam van de class: WsCoreResponseUrlDataRepository te maken?

Reply:
I have renamed this class and the other repository classes as well.